packages/types/src/extrinsic/signedExtensions/index.ts

```diff
@@ -9,13 +9,14 @@ import { polkadot } from './polkadot';
 import { shell } from './shell';
 import { statemint } from './statemint';
 import { substrate } from './substrate';
+import { encointer } from './encointer';
 
 // A mapping of the known signed extensions to the extra fields that they
 // contain. Unlike in the actual extensions, we define the extra fields not
 // as a Tuple, but rather as a struct so they can be named. These will be
 // expanded into the various fields when added to the payload (we only
 // support V4 onwards with these, V3 and earlier are deemed fixed))
-export const allExtensions: ExtDef = objectSpread({}, substrate, polkadot, shell, statemint);
+export const allExtensions: ExtDef = objectSpread({}, substrate, polkadot, shell, statemint, encointer);
```

Comment:
In this case, this extension would override the existing `ChargeAssetTxPayment` for statemint, which means it will break usage on statemint.

The idea with these are that they map to the default pallets for Substrate & Polkadot. Anything chain-specific needs to be supplied as part of the chain types. For instance as used in the apps UI.

Reply:
I see, thanks for explaining. However, I would like to request #4197 once again because this makes many the polkadot-js/api powered products hard to use for us, e.g. subquery because it can't interpret our extrinsics, and it does not offer this kind of customization without entering the source code.

Reply:
The linked issue is in the queue - so yea, will be done at some point.

Reply:
Ok, thanks. I will close the PR then.